src/reducers/Entities/NameReducer.js

```diff
@@ -35,6 +38,20 @@ export const NameReducer = (state = initialState(), action) => {
 
       return { ...state, editing: newName };
     }
+
```

Comment:
nit: The `.SAVE` action had me a little confused - it's more of a "select" or "start using this one" kind of action, where a "SAVE" for me implies persistently storing

Reply:
me too - we have `save` elsewhere for this kind of action so I went for consistency.
I'll change though, thanks.